source/architecture.md

```diff
@@ -1,33 +1,37 @@
 
 > For example, [`openfisca-france`](https://github.com/openfisca/openfisca-france) models French law.
 
-## Extensions packages
+## The Extensions Packages
 
-Extensions add capabilities to a country package while maintaining situation description compatibility by adding or redefining parameters and variables, but not entities.
+Extensions add functionalities to a country package by adding or redefining parameters and variables, but not entities while maintaining situation description compatibility.
 
-This architecture enables subcommunities to maintain subsets of the rules of a country on which they have authority, or which are too specific to be of interest to the wider OpenFisca French community.
+<!--- Needs clarity here----->
+This architecture enables sub-communities to maintain subsets of the rules of a country on which they have the authority, or are too specific to be of interest to the border OpenFisca community.
 
 > For example, [`openfisca-paris`](https://github.com/openfisca/openfisca-paris) extends `openfisca-france` with benefits specific to the town of Paris and is maintained by the Paris City Council.
 
-## Core
+## The Core
 
-OpenFisca Core provides the API, domain-specific language (DSL) and testing tools with which country and extension packages are built.
+OpenFisca Core provides the API, domain-specific language (DSL) and testing tools. The country and extension packages are constructed based on the Core.
 
-It defines the [Python API](openfisca-python-api/index.md) through which the country packages can be queried.
+The core defines the [Python API](openfisca-python-api/index.md) through which the country packages can be queried.
 
 The Core also includes an optional [web API](openfisca-web-api/index.md), which makes it possible to expose [Parameters](key-concepts/parameters.md) and calculate [Variables](key-concepts/variables.md) over HTTP and JSON.
 
 ## Templates
 
-Since all country and extension packages are maintained by different communities and depend on legal and cultural contexts, OpenFisca provides two packages aimed at demonstrating its capabilities independently: [`country-template`](https://github.com/openfisca/country-template/) and [`extension-template`](https://github.com/openfisca/extension-template/). They can be used to bootstrap new country or extension packages.
-
+Country and extension packages are maintained by different communities and depend on legal and cultural contexts. OpenFisca provides two templates aimed at demonstrating its capabilities independently:
+[`country-template`](https://github.com/openfisca/country-template/)  
+[`extension-template`](https://github.com/openfisca/extension-template/).
+They can be bootstrapped with a new country or extension package.
+<!--Sentence unclear-->
 This documentation uses these packages throughout.
```

Comment:
Does the above line mean that " throughout the openfisca documentation the packages are used ''? or something else?

Reply:
It's not clear what "they" refers to

Reply:
Ah got it! then we can rephrase it to " Code examples throughout the OpenFisca documentation use both country-template and extension-template as a base"😁

Reply:
That's perfect! <3